pkg/api/v1/helper/helpers.go

```diff
@@ -498,3 +498,36 @@ func PersistentVolumeClaimHasClass(claim *v1.PersistentVolumeClaim) bool {
 
 	return false
 }
+
+// NodeConditionEqual returns true if all effective types ("Status" is true) equals;
+// otherwise, returns false.
+func NodeConditionEqual(c1 []v1.NodeCondition, c2 []v1.NodeCondition) bool {
+	if len(c1) == 0 && len(c2) == 0 {
+		return true
+	}
+
+	if len(c1) != 0 || len(c2) != 0 {
```

Comment:
wow, this's used when filtered status is `true` condition before. It should be removed.